politeiawww/politeiawww.go

```diff
@@ -122,6 +123,9 @@ type politeiawww struct {
 
 	// cmsDB is only used during cmswww mode
 	cmsDB cmsdatabase.Database
+
+	// cron
+	cron *cron.Cron
```

Comment:
Could we change this to something like:

```
	// Following entries are used only during cmswww mode
	cmsDB cmsdatabase.Database
	cron *cron.Cron
```

Just so that we keep track of what is cms only.